packages/components/.stylelintcache

```diff
@@ -0,0 +1 @@
mtime":1657268332507,"hashOfConfig":"72"},{"size":6974,"mtime":1651679898792,"hashOfConfig":"72"},{"size":617,"mtime":1657268332507,"hashOfConfig":"72"},{"size":1015,"mtime":1657268332508,"hashOfConfig":"72"},{"size":4602,"mtime":1657268332508,"hashOfConfig":"72"},{"size":9155,"mtime":1657268332509,"hashOfConfig":"72"},{"size":1152,"mtime":1657268332509,"hashOfConfig":"72"},{"size":8470,"mtime":1657191154792,"hashOfConfig":"72"},{"size":6608,"mtime":1657268332509,"hashOfConfig":"72"},{"size":35,"mtime":1651679898793,"hashOfConfig":"72"},{"size":2754,"mtime":1657268332509,"hashOfConfig":"72"},{"size":1170,"mtime":1657268332509,"hashOfConfig":"72"},{"size":890,"mtime":1657268332510,"hashOfConfig":"72"},{"size":1841,"mtime":1657268332510,"hashOfConfig":"72"},{"size":215,"mtime":1657268332511,"hashOfConfig":"72"},{"size":675,"mtime":1657268332511,"hashOfConfig":"72"},{"size":2969,"mtime":1657268332511,"hashOfConfig":"72"},{"size":9435,"mtime":1655370336257,"hashOfConfig":"72"},{"size":1626,"mtime":1657268332515,"hashOfConfig":"72"},{"size":330,"mtime":1657194056238,"hashOfConfig":"72"},{"size":860,"mtime":1653379036317,"hashOfConfig":"72"},{"size":2616,"mtime":1657268332516,"hashOfConfig":"72"},{"size":919,"mtime":1657268332517,"hashOfConfig":"72"},{"size":1102,"mtime":1651679898796,"hashOfConfig":"72"},{"size":3318,"mtime":1657268332519,"hashOfConfig":"72"},{"size":1726,"mtime":1657268332511,"hashOfConfig":"72"},{"size":1189,"mtime":1651679898794,"hashOfConfig":"72"},{"size":1800,"mtime":1657268332519,"hashOfConfig":"72"},{"size":508,"mtime":1657194056233,"hashOfConfig":"72"},{"size":1458,"mtime":1657194056233,"hashOfConfig":"72"},{"size":1381,"mtime":1657194056234,"hashOfConfig":"72"},{"size":2096,"mtime":1657268332512,"hashOfConfig":"72"},{"size":1613,"mtime":1657194056234,"hashOfConfig":"72"},{"size":903,"mtime":1657194056235,"hashOfConfig":"72"},{"size":1451,"mtime":1657194056235,"hashOfConfig":"72"},{"size":3089,"mtime":1657268332513,"hashOfConfig":"72"},{"size":584,"mtime":1657194056235,"hashOfConfig":"72"},{"size":1157,"mtime":1657268332513,"hashOfConfig":"72"},{"size":300,"mtime":1657268332514,"hashOfConfig":"72"},{"size":1406,"mtime":1651679898795,"hashOfConfig":"72"},{"size":474,"mtime":1657268332514,"hashOfConfig":"72"},{"size":737,"mtime":1657268332516,"hashOfConfig":"72"},{"size":1357,"mtime":1657268332516,"hashOfConfig":"72"},{"size":500,"mtime":1651679898796,"hashOfConfig":"72"},{"size":849,"mtime":1657268332516,"hashOfConfig":"72"},{"size":1692,"mtime":1657268332517,"hashOfConfig":"72"},{"size":5331,"mtime":1657268332517,"hashOfConfig":"72"},{"size":3038,"mtime":1657268332517,"hashOfConfig":"72"},{"size":824,"mtime":1657268332518,"hashOfConfig":"72"},{"size":1060,"mtime":1651679898795,"hashOfConfig":"72"},{"size":514,"mtime":1657268332515,"hashOfConfig":"72"},{"size":1047,"mtime":1657268332515,"hashOfConfig":"72"},{"size":671,"mtime":1657268332518,"hashOfConfig":"72"},{"size":1073,"mtime":1657268332519,"hashOfConfig":"72"},"1y6arci"]
```

Comment:
please remove stylelint cache files, they are in gitignore

Reply:
fixed